.circleci/config.yml

```diff
@@ -2,6 +2,21 @@
 # See: https://circleci.com/docs/configuration-reference
 version: 2.1
 
+aliases:
+  - &on_master
+    context: spaceshard
+    filters:
+      branches:
+        only:
+          - master
```

Comment:
master or main?

Reply:
Thanks, addressed in: https://github.com/0xSpaceShard/starknet-devnet-rs/pull/151/commits/8d0a2cebe47f96dc67bf75f018afbf166d25b61a